packages/shared-state-async-node/Makefile

```diff
@@ -0,0 +1,81 @@
+# Shared State
+#
+# Copyright (c) 2023  Javier Jorge <jjorge@inti.gob.ar>
+# Copyright (c) 2023  Instituto Nacional de Tecnología Industrial
+# Copyright (C) 2023  Asociación Civil Altermundi <info@altermundi.net>
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the
+# Free Software Foundation, version 3.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+include $(TOPDIR)/rules.mk
+
+GIT_COMMIT_DATE:=$(shell git log -n 1 --pretty=%ad --date=short . )
+GIT_COMMIT_TSTAMP:=$(shell git log -n 1 --pretty=%at . )
+
+PKG_NAME:=shared-state-async-node
+PKG_VERSION=$(GIT_COMMIT_DATE)-$(GIT_COMMIT_TSTAMP)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/libremesh/shared-state-async-node.git
+PKG_SOURCE_VERSION:= 691c4412a84bb8142c73b4f5ff319f025b3973f7
+PKG_MAINTAINER:=Javier Jorge <javierbrk@gmail.com>
+PKG_LICENSE:=AGPL-3.0
+
+HOST_BUILD_PREFIX:=$(STAGING_DIR_HOST)
+
+# Source settings (i.e. where to find the source codes)
+# This is a custom variable, used below 
+# disable git package source and enable Build/Prepare line 29-30
+# SOURCE_DIR:=
```

Comment:
This block should be removed as source tree override should be used for that purpose as explained by Jow here https://forum.archive.openwrt.org/viewtopic.php?id=46916